<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>treeViewMainWindow</class>
 <widget class="QMainWindow" name="treeViewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../automatehandler.qrc">
    <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="nodeTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="autoExpandDelay">
       <number>20</number>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="collapseAllBtn">
        <property name="text">
         <string>&amp;CollapseAll</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QPushButton" name="expandAllBtn">
        <property name="text">
         <string>&amp;ExpandAll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="addConnectionBtn">
        <property name="text">
         <string>Add co&amp;nnection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="delConnectionBtn">
        <property name="text">
         <string>Del C&amp;onnection</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="addNodeBtn">
        <property name="text">
         <string>&amp;Add Node</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="delNodeBtn">
        <property name="text">
         <string>&amp;Del Node</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../automatehandler.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>expandAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>nodeTreeView</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collapseAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>nodeTreeView</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
